Gemfile

```diff
@@ -1,7 +1,13 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
 # Specify your gem's dependencies in ocso_scraper.gemspec
 gemspec
 
-gem "rake", "~> 12.0"
```

Comment:
why the removal of rspec and rake gems?

Reply:
It's still there just rubocop changed it to double quotes